tests/acceptance/features/apiNotification/spaceNotification.feature

```diff
@@ -96,10 +96,7 @@ Feature: Notification
                     ]
                   },
                   "id": {
-                    "type": "string",
-                    "enim": [
```

Comment:
here was typo `enim` and I guess this test was green

Reply:
the tests passed because of `type: string` and `enim` was ignored because enim is not a valid schema validator

Reply:
now, I have checked for entries that are not schema validator